pages/home.html

```diff
@@ -1,15 +1,21 @@
 <section id="main_content" class="inner">
 
+  <h2>
+    <a id="main-idea" class="anchor" href="#main-idea" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Ideas
```

Comment:
The structure of the page is divided to 5 main sections:

- Section 1 with the video, bite-sized, interactive, adaptive
- Section 2: Features
- Section 3: Requirements
- Section 4: Contributing to Oppia
- Section 5: Helpful Links (should we change it to sentence capitalization 'Helpful links'?)

I observe that each section has its own heading, except the first one where we talk about the creative ideas behind Oppia. We should give it a heading too. Because once readers see a heading, their mind automatically predict the content of that section, it'll help them not to get lost in the documentation.